### ISSUE TITLE
[CoE Starter Kit - BUG] HELPER - Maker Check fails 95 % of the time

### ISSUE THREAD

**Issue by kimjensen90** [microsoft/coe-starter-kit#3599]
### Describe the issue

Hi, 

First of all, thanks for all your efforts on the CoE Starter Kit! 

In my organization we have persistent issue with the "HELPER - Maker Check Flow", which is quite essential to all the "Admin | Sync" Flows. It seems to fail in 95 % of its runs with the error: "admin_maker With Id = xxxx-xxxx-xxxx Does Not Exist". If I try to search for the record in Dataverse, I can confirm that it doesn't exists. 

I would expect the manager to be created or the field admin_managerid to be updated. Alternatively the step could be skipped in case of errors. As mentioned above the error causes many issues for the entire "Admin | Sync" process, as the Flows don't receive any response from "HELPER - Maker Check". I do understand that a manager isn't necessarily a "Maker". 

I have tried to enable "FullInventory", but it doesn't seem to have any effect. 

In addition, I would expect our admin_maker table to contain more makers than it currently does. Previously we tried to delete all of them and have them re-created through the sync process, but it doesn't seem to help.  

I hope you can help me resolving this issue. 

Thank you very much! 

![image](https://user-images.githubusercontent.com/104207653/188844844-9a4b33f5-f8b3-4d11-a627-97ff81bfd57e.png)

### Expected Behavior

I would expect the manager to be created or the field admin_managerid to be updated. Alternatively the step could be skipped in case of errors. As mentioned above the error causes many issues for the entire "Admin | Sync" process, as the Flows don't receive any response from "HELPER - Maker Check". I do understand that a manager isn't necessarily a "Maker". 

In short, I would just like the Flow to succeed. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.41

### What app or flow are you having the issue with?

HELPER - Maker Check

### Steps To Reproduce

Starting the "Sync-process" triggers this issue.

**Comment by Jenefer-Monroe** [comment]
Hello. I believe we had and fixed this bug in the past. Can you please check for unmanaged layers on the flow? 
If you have one, remove it. [Remove unmanaged layers from inventory flows](https://docs.microsoft.com/power-platform/guidance/coe/after-setup#remove-unmanaged-layers-from-inventory-flows)

Let me know!

**Comment by kimjensen90** [comment]
Hi @Jenefer-Monroe, 

Thank you so much for the swift reply - that definitely seems to be the culprit. All removed and now the flow succeeds. 

Cheers! 
![image](https://user-images.githubusercontent.com/104207653/188890008-b2805d90-3a43-4b02-bdb7-8d8d2a005f03.png)

**Comment by Jenefer-Monroe** [comment]
wonderful. Thank you for letting me know!
